lang/user/id.xml

```diff
@@ -561,25 +563,25 @@ Silakan download dan install firmware PS Vita terakhir.</no_modules>
 		</core>
 		<cpu>
 			<cpu_backend>Bagian Belakang CPU</cpu_backend>
-			<select_cpu_backend>Pilih backend CPU pilihan Kamu.</select_cpu_backend>
-			<cpu_opt>Aktifkan pengoptimalan</cpu_opt>
+			<select_cpu_backend>Pilih perender backend pilihan kamu.</select_cpu_backend>
+			<cpu_opt>Aktifkan Pengoptimalan</cpu_opt>
 			<cpu_opt_description>Centang kotak untuk mengaktifkan optimasi CPU JIT tambahan.</cpu_opt_description>
 		</cpu>
 		<gpu>
 			<backend_renderer>Bagian Belakang Perender</backend_renderer>
-			<select_backend_renderer>Pilih perender backend pilihan Kamu.</select_backend_renderer>
+			<select_backend_renderer>Pilih perender backend pilihan kamu.</select_backend_renderer>
 			<gpu>GPU (Harus ulang lagi aplikasinya)</gpu>
 			<select_gpu>Pilih GPU Vita3K yang harus dijalankan.</select_gpu>
 			<renderer_accuracy>Perender Akurasi</renderer_accuracy>
 			<v_sync_description>Menonaktifkan V-Sync dapat memperbaiki masalah kecepatan di beberapa game.
 Disarankan untuk tetap mengaktifkannya untuk menghindari robekan visual.</v_sync_description>
-			<disable_surface_sync>Nonaktifkan sinkronisasi permukaan</disable_surface_sync>
+			<disable_surface_sync>Nonaktifkan Edit Sinkronisasi</disable_surface_sync>
 			<surface_sync_description>Speed hack, centang kotak untuk menonaktifkan sinkronisasi permukaan antara CPU dan GPU.
-Sinkronisasi permukaan diperlukan oleh beberapa game.
+Edit Sinkronisasi diperlukan oleh beberapa game.
 Memberikan peningkatan kinerja yang besar jika dinonaktifkan (khususnya saat peningkatan skala aktif).</surface_sync_description>
-			<screen_filter>Filter layar</screen_filter>
+			<screen_filter>Penyesuaian Layar</screen_filter>
 			<screen_filter_description>Setel filter pasca-pemrosesan untuk diterapkan.</screen_filter_description>
-			<internal_resolution_upscaling>Internal Resolusi Upscaling</internal_resolution_upscaling>
+			<internal_resolution_upscaling>Internal Reso Pelukisan</internal_resolution_upscaling>
```

Comment:
@darkash please check this.

Reply:
> @darkash could you check this?

This one is ok. The old one feels off since pelukisan it translates to "internal resolution painting (?)". We discussed and finally chose this one as the most commonly understood.